src/components/CardZoom/index.js

```diff
@@ -21,14 +21,14 @@ export default React.memo(function CardZoom(props) {
       document.removeEventListener('keydown', handleESC)
       document.documentElement.style.overflowY = ''
     }
-  }, [handleESC, props.cardId])
+  }, [handleESC, props.card])
 
-  return props.cardId ? (
+  return props.card ? (
     <div className='CardZoom__overlay' onClick={props.close} data-testid='zoom'>
       <div className='CardZoom__wrapper'>
         <Card
           {...resolveCardForLevel({
-            id: props.cardId,
+            id: props.card.id,
```

Comment:
I don’t think this file needs to be modified at all. What if `resolveCardForLevel` just took into account that `id` can contain `_x` and breaks it before the database lookup? This way this component could keep working like it always had.